packages/singleton-manager/src/SingletonManagerClass.js

```diff
@@ -1,9 +1,18 @@
 const sym = Symbol.for('lion::SingletonManagerClassStorage');
 
+/**
+ * Allow compatibility with node-js (for ssr).
+ * In the future, we can just use globalThis directly
+ * (for now, we're backwards compatible with browsers that still only use window, since we don't know all contexts singleton-manager is used in).
+ */
+// eslint-disable-next-line no-undef
+const globalThisOrWindow = globalThis || window;
 export class SingletonManagerClass {
   constructor() {
     /** @protected */
-    this._map = window[sym] ? window[sym] : (window[sym] = new Map());
```

Comment:
Should we provide `globalThisOrWindow ` as a constructor parameter and then update unit tests so that they all run for both `window` and `globalThis`?

Reply:
Was indeed thinking of how we can test this. The right thing to do would probably be to run all the current test in a node runtime (via mocha) to ensure we can use singletonManager with node.
I would rather focus on that (at a later point), then providing globalThis to the constructor.

At some point (when we add ssr support for lion) we need to test all our code in node as well. But since this change was so small and straightforward (`globalThis` with a fallback to `window`) I think we could get away w/o a node test for now.

Reply:
Update: added a commit to run the same tests in node as well :)